database/config.js

```diff
@@ -1,94 +1,28 @@
 /* eslint-disable no-console */
-const mongoose = require('mongoose');
+const { Client } = require('pg');
 
-const db = mongoose.connection;
-
-mongoose.connect('mongodb://localhost/mediaDb', { useNewUrlParser: true });
-
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', () => {
-  console.log('connected to db');
-});
-
-const gameSchema = new mongoose.Schema({
-  proxyId: Number,
-  summary: {
-    title: String,
-    splash: String,
-    description: String,
-    reviews: { general: String, total: Number },
-    releaseDate: String,
-    developer: String,
-    publisher: String,
-    tags: [String],
-    percentage: Number,
-  },
-  media: {
-    video: [{ video: String, thumbnail: String }],
-    images: [String],
-  },
+const client = new Client({
+  user: 'postgres',
+  password: 'postgres',
+  port: 5432,
+  database: 'games',
 });
 
-const Game = mongoose.model('Game', gameSchema);
-
-module.exports.addNewGame = (game, cb) => {
-  const newGame = new Game(game);
-  // eslint-disable-next-line no-shadow
-  newGame.save((err, newGame) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(null, newGame);
-    }
-  });
-};
 
-module.exports.getGame = (game, cb) => {
-  Game.find({ proxyId: game.proxyId }, (err, data) => {
+const getGame = (id, callback) => {
+  client.query(`SELECT * FROM gamestable WHERE id=${id}`, (err, results) => {
     if (err) {
-      cb(err);
+      callback(err);
     } else {
-      cb(null, data);
+      callback(null, results);
     }
   });
 };
 
-module.exports.postGame = (game, cb) => {
-  Game.insert(game, (err, result) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(null, result);
-    }
-  });
-};
+client.connect()
+  .then(() => console.log('connected!!!!!!!!!!!!!'))
+  .catch((err) => console.error(err));
 
-module.exports.deleteGame = (obj, cb) => {
-  Game.deleteOne(obj, (err, result) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(result);
-    }
-  });
-};
-
-module.exports.putGame = (search, obj, cb) => {
-  Game.updateOne(search, obj, (err, result) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(null, result);
-    }
-  });
-};
-
-module.exports.deleteGame = (obj, cb) => {
-  Game.deleteOne(obj, (err, result) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(null, result);
-    }
-  });
+module.exports = {
```

Comment:
it looks like you've got some module.exports sprinkled throughout the file, might be easier for future you and others to read if you do it all the same way (put all functions into module.exports, or export them individually), overall looks good though

Reply:
Thank you! that's so weird, those aren't in vscode on my machine. I'll have to look into that.